packages/client/tests/functional/typed-sql/mysql-scalars/test.ts

```diff
@@ -81,6 +81,11 @@ testMatrix.setupTestSuite(
       expect(result[0].id).toEqual(id)
     })
 
+    test('string - input with doc comment', async () => {
```

Comment:
Query: Is this test sufficient for the linked PR?  @Weakky  may know more about this.